@xen-orchestra/web/src/jobs/vm-create.jobs.ts

```diff
@@ -0,0 +1,14 @@
+import { useFetch } from '@vueuse/core'
+
+export async function createVM(payload: Record<string, any>, poolId: string) {
```

Comment:
```suggestion
export async function createVM(payload: Record<string, any>, poolId: XoPool['id']) {
```
We could improve the _payload_ typing and use the Job System